test/module/irohad/ametsuchi/wsv_query_command_test.cpp

```diff
@@ -190,6 +187,19 @@ CREATE TABLE IF NOT EXISTS account_has_grantable_permissions (
       ASSERT_EQ(account.json_data, acc.value().json_data);
     }
 
+    /**
+     * @given inserted role, domain, account
+     * @when update account json data
+     * @then get account and check json data is the same
+     */
+    TEST_F(AccountTest, UpdateAccountWithJSONData) {
+      ASSERT_TRUE(command->insertAccount(account));
+      ASSERT_TRUE(command->setAccountKV(account.account_id, "id", "val"));
+      auto acc = query->getAccount(account.account_id);
+      ASSERT_TRUE(acc.has_value());
+      ASSERT_EQ(R"({"id": "val", "key": "value"})", acc.value().json_data);
```

Comment:
JSON does not provide order on fields. Better to compare fields directly not raw strings.